pwiz_tools/Skyline/TestTutorial/GroupedStudiesTutorialTest.cs

```diff
@@ -202,19 +200,23 @@ protected override void DoTest()
                 OkDialog(peptideSettingsUI2, peptideSettingsUI2.OkDialog);
             }
 
-            RunUI(() => SkylineWindow.SaveDocument(GetOcRawTestPath("OC-study.sky")));
+            RunUI(() => SkylineWindow.SaveDocument(GetTestPath("OC-study.sky")));
 
             // Importing Data
             ImportResultsFiles(GetOcRawTestPath(), ExtAbWiff,
                 IsFullData ? "R201217" : "R201217_plasma_revision_A", true);
             WaitForCondition(5*60*1000, () =>
                 SkylineWindow.Document.Settings.HasResults && SkylineWindow.Document.Settings.MeasuredResults.IsLoaded);
+            WaitForDocumentLoaded();

```

Comment:
These WaitForDocumentLoaded() changes look like the only meaningful changes here to me. Though, mildly surprising that the results loading is not the slowest background loader.